### ISSUE TITLE
hyperv: PermissionDenied: (:) [Start-VM], VirtualizationException

### ISSUE THREAD

**Issue by AAATechGuy** [kubernetes/minikube#12555]
After installed minikube, I tried to start it with minikube start --vm-driver=hyperv. But got the following error:

```
E0922 15:57:12.232773   16868 main.go:126] libmachine: [stderr =====>] : Hyper-V\Start-VM : 'minikube' failed to start. (Virtual machine ID EC3973B7-2291-437D-9BD3-4BC38FCDE770)
At line:1 char:1
+ Hyper-V\Start-VM minikube
+ ~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (:) [Start-VM], VirtualizationException
    + FullyQualifiedErrorId : AccessDenied,Microsoft.HyperV.PowerShell.Commands.StartVM
 
W0922 15:57:25.748411   16868 out.go:242] X Exiting due to GUEST_NOT_FOUND: Failed to start host: Error loading existing host. Please try running [minikube delete], then run [minikube start] again.: filestore "minikube": open C:\Users\cloudtest\.minikube\machines\minikube\config.json: The system cannot find the file specified.
```

Attaching logs: [logs.txt](https://github.com/kubernetes/minikube/files/7214292/logs.txt)

Tried `minikube delete` and start again, doesn't work. Running command as Admin. 

Duplicate of https://github.com/kubernetes/minikube/issues/6918, but I can't reopen it.

**Comment by spowelljr** [comment]
Hi @AAATechGuy, thanks for reporting your issue with minikube!

Are you able to start a VM from PowerShell yourself? For example (don't for get to run as admin):
```
New-VM -Name Win10VM -MemoryStartupBytes 4GB -BootDevice VHD -VHDPath .\VMs\Win10.vhdx -Path .\VMData -Generation 2 -Switch ExternalSwitch

Start-VM -Name Win10VM
```

**Comment by AAATechGuy** [comment]
After running commands on Hyper-V VM, the VM seems to start-up. However, stuck in Running phase. 

Please note, this is nested-virtualization enabled AzureVM, running on windows-server-2019. 

```
Write-Host 'checking if RSAT-Hyper-V-Tools is installed..';
if((Get-WindowsFeature RSAT-Hyper-V-Tools).InstallState -ne 'Installed') {
  Write-Host 'RSAT-Hyper-V-Tools not found. Installing..';
  Install-WindowsFeature -Name RSAT-Hyper-V-Tools -IncludeAllSubFeature
}

$externalNetworkName = (Get-NetAdapter).Name | where { $_.StartsWith('Ethernet') };
New-VMSwitch -name ExternalSwitch  -NetAdapterName $externalNetworkName -AllowManagementOS $true

Get-Service vmms | Restart-Service

choco install minikube

minikube start --alsologtostderr -v=2
```

Here's the screenshot... 

![image](https://user-images.githubusercontent.com/3484074/134786218-fde3ba45-6d8f-4079-9fc5-31b0b5c23c91.png)

![image](https://user-images.githubusercontent.com/3484074/134786224-56fd281e-46f1-4a88-a83a-48b0d6537201.png)

**Comment by spowelljr** [comment]
Hi @AAATechGuy, thanks for the update, can you please run `minikube logs --file=logs.txt` and upload the newly generated log file.

**Comment by spowelljr** [comment]
Hi @AAATechGuy, we haven't heard back from you, do you still have this issue?
There isn't enough information in this issue to make it actionable, and a long enough duration has passed, so this issue is likely difficult to replicate.

I will close this issue for now but feel free to reopen when you feel ready to provide more information.

**Comment by AAATechGuy** [comment]
_We still have issue, we ended up creating a linux VM with kubernetes on it to unblock temporarily. Issue still exists. I'll get back with more details later._
